test/Microsoft.AspNetCore.Razor.Language.Test/Extensions/FunctionsDirectivePassTest.cs

```diff
@@ -69,7 +69,7 @@ public void Execute_MovesStatementsToClassLevel()
                 node => CSharpCode(" var value = true; ", node));
 
             var method = (MethodDeclarationIntermediateNode)@class.Children[0];
-            Assert.Empty(method.Children);
+            SingleChild<DirectiveIntermediateNode>(method);
```

Comment:
Validate this like you do the others bits in this test. It's now an add test, not a move.